pkg/db/transactions.go

```diff
@@ -62,6 +81,11 @@ func WithTransactionT[T any](h *DBHandler, ctx context.Context, readonly bool, f
 
 // WithTransactionMultipleEntriesT is the same as WithTransaction, but you can also return and array of data, not just the error.
 func WithTransactionMultipleEntriesT[T any](h *DBHandler, ctx context.Context, readonly bool, f DBFunctionMultipleEntriesT[T]) ([]T, error) {
+	return WithTransactionMultipleEntriesRetryT(h, ctx, DefaultNumRetries, readonly, f)
+}
+
+// WithTransactionMultipleEntriesRetryT also supports retries
+func WithTransactionMultipleEntriesRetryT[T any](h *DBHandler, ctx context.Context, maxRetries uint, readonly bool, f DBFunctionMultipleEntriesT[T]) ([]T, error) {
```

Comment:
I think we should set the `maxRetries` to `uint8` so that we don't just get stuck here forever if a large retries number is passed.
Or we can can check if `maxRetries` is greater than `DefaultNumRetries` we pass the `DefaultNumRetries`.

Reply:
done